sources/packages/api/src/route-controllers/institution/institution.aest.controller.ts

```diff
@@ -75,12 +75,16 @@ export class InstitutionAESTController extends BaseController {
       legalName: eachInstitution.legalOperatingName,
       operatingName: eachInstitution.operatingName,
       address: {
-        addressLine1: eachInstitution.institutionAddress.addressLine1,
-        addressLine2: eachInstitution.institutionAddress.addressLine2,
-        city: eachInstitution.institutionAddress.city,
-        provinceState: eachInstitution.institutionAddress.provinceState,
-        country: eachInstitution.institutionAddress.country,
-        postalCode: eachInstitution.institutionAddress.postalCode,
+        addressLine1:
+          eachInstitution.institutionAddress.mailingAddress?.addressLine1,
```

Comment:
I believe that `mailingAddress` will always be present right? Do we need null checks?

Reply:
I guess, you are right @andrewsignori-aot in the institution creation formio forms, those are mandatory fields. I think, this was introduced sometimes recently, as I see in Dev, some institutions does not have mailing address. We can safely remove the null checks. 
Also instead of repeating eachInstitution.institutionAddress.mailingAddress, we can create a constant called as mailing address inside the method and use it inside the address block.

Reply:
as discussed the null check is added for old data,

Reply:
Can we please create a constant for eachInstitution.institutionAddress.mailingAddress, instead of repeating them?